Microsoft.Toolkit.Uwp.UI.Controls.DataGrid/DataGrid/DataGridComboBoxColumn.cs

```diff
@@ -205,11 +214,11 @@ protected override FrameworkElement GenerateEditingElement(DataGridCell cell, ob
 
             if (dataItem != null)
             {
-                var value = dataItem.GetType().GetProperty(Binding.Path.Path).GetValue(dataItem);
+                var value = dataItem.GetType().GetProperty(this.Binding.Path.Path).GetValue(dataItem);
 
-               var selection = !string.IsNullOrEmpty(DisplayMemberPath)
-                    ? ItemsSource?.Cast<object>().FirstOrDefault(x => x.GetType().GetProperty(Binding.Path.Path).GetValue(x).Equals(value))
-                    : ItemsSource?.Cast<object>().FirstOrDefault(x => x.Equals(value));
+               var selection = !string.IsNullOrEmpty(this.DisplayMemberPath)
+                    ? this.ItemsSource?.Cast<object>().FirstOrDefault(x => x.GetType().GetProperty(this.Binding.Path.Path).GetValue(x).Equals(value))
```

Comment:
per the [C# coding guidelines](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md) we should avoid using `this.`

Reply:
@MSRegisB, could you update the PR to remove the `this.`?